Dockerfile

```diff
@@ -9,7 +9,7 @@
 # docker run -v `pwd`:/go/src/github.com/docker/docker --privileged -i -t docker bash
 #
 # # Run the test suite:
-# docker run -e DOCKER_GITCOMMIT=foo --privileged docker hack/make.sh test-unit test-integration-cli test-docker-py
```

Comment:
@dnephin I understand that we're deprecating test-integration-cli, but I'm not sure I would like people to not run them just yet.